Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="30" judge-id="31" status="SE" scoring="ACM" run-tests="1" contest-id="3205" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>22e3628d-8022-4621-b7cb-c6e4f44b62e3</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="288" real-time="302" max-memory-used="18698240" >
  <program-stats-str>{ utime=243, stime=45, ptime=288, rtime=302, maxvsz=18698240, maxrss=19349504, nvcsw=1, nivcsw=32 }</program-stats-str>
      <input size="12">5 -4 3 -2 1
</input>
      <output size="0"></output>
      <stderr size="276">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000030&quot;, line 7, in &lt;module&gt;
    if a[i]&lt;1:
IndexError: list index out of range
</stderr>
    </test>
  </tests>
</testing-report>
